DUI3-DX/Converters/Revit/Speckle.Converters.RevitShared/ToSpeckle/CurveConversionToSpeckle.cs

```diff
@@ -0,0 +1,34 @@
+using Speckle.Converters.Common.Objects;
+using Speckle.Converters.Common;
+using Objects;
+
+namespace Speckle.Converters.RevitShared.ToSpeckle;
+
+[NameAndRankValue(nameof(DB.Curve), 0)]
+public class CurveConversionToSpeckle : BaseConversionToSpeckle<DB.Curve, ICurve>
+{
+  private readonly IRawConversion<DB.Line, SOG.Line> _lineConverter;
+
+  //private readonly IRawConversion<DB.Arc, SOG.Line> _arcConverter;
```

Comment:
Commented out section?

Reply:
those conversion don't exist yet. Was thinking we could uncomment them as we implement them. Or I could just remove the comments, it wouldn't be much work to redo

Reply:
Add POC comments on all these things so we can search for them later 😉

Reply:
Let's add a POC: comment here so we can hoover it up later